sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/Optimizer.scala

```diff
@@ -732,11 +732,12 @@ object SimplifyCasts extends Rule[LogicalPlan] {
 }
 
 /**
- * Removes [[UnaryPositive]] identify function
+ * Removes [[UnaryPositive]] identify function and [[PromotePrecision]] wrapper.
```

Comment:
`Removes identify functions` ? Just in case we need to remove more functions like these.